site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -50,14 +50,104 @@ public function editGradeableRequest($gradeable_id, $nav_tab = 0) {
         'electronic_bulk' => 'TA/Instructor will (bulk) upload scanned .pdf for online manual grading'
     ];
 
+    /**
+     * @Route("/api/{_semester}/{_course}/{gradeable_id}/download", methods={"GET"})
+     * @Route("/courses/{_semester}/{_course}/{gradeable_id}/download", methods={"GET"})
+     */
+    public function downloadJson(string $gradeable_id) {
```

Comment:
This should fix the PHPStan error:
```suggestion
    public function downloadJson(string $gradeable_id): void {
```